	yeah, right. Disabling everything with echo-statements is really useful.
--- iproute2-2.6.34.orig/tc/Makefile	2010-05-19 17:32:43.000000000 +0200
+++ iproute2-2.6.34/tc/Makefile	2011-01-03 18:28:02.980563959 +0100
@@ -99,10 +99,11 @@ libtc.a: $(TCLIB)
 	$(AR) rcs $@ $(TCLIB)
 
 install: all
-	echo mkdir -p $(MODDESTDIR)
-	echo install -m 0755 tc $(DESTDIR)$(SBINDIR)
+	mkdir -p $(MODDESTDIR)
+	mkdir -p $(DESTDIR)$(SBINDIR)
+	install -m 0755 tc $(DESTDIR)$(SBINDIR)
 	for i in $(TCSO); \
-	do echo install -m 755 $$i $(MODDESTDIR); \
+	do install -m 755 $$i $(MODDESTDIR); \
 	done
 	if [ ! -f $(MODDESTDIR)/m_ipt.so ]; then \
 	if [ -f $(MODDESTDIR)/m_xt.so ]; \
